src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/InputParenPadForSynchronized.java

```diff
@@ -0,0 +1,34 @@
+/*
+ParenPad
+option = (default)nospace
+tokens = LITERAL_SYNCHRONIZED, CTOR_DEF, CTOR_CALL, LITERAL_IF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.parenpad;
+
+/**
+ *
+ * Thread-safe Singleton class.
+ * The instance is lazily initialized and thus needs synchronization
+ * mechanism.
+ *
+ */
```

Comment:
Please remove all unrelated to problem code.
We need to minimize code in input to keep only what matters.

Reply:
done